README.md

```diff
@@ -1,2 +1,130 @@
+    </a>
+    <a href="https://www.npmjs.com/package/virtual-google-assistant">
+        <img alt="NPM Version" class="badge" src="https://img.shields.io/npm/v/virtual-google-assistant.svg">
+    </a>
+    <a href="https://gitter.im/bespoken/bst?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge">
+        <img alt="Read The Docs" class="badge" src="https://badges.gitter.im/bespoken/bst.svg">
+    </a>
+</p>
+
+
+# Virtual Google Assistant
+Virtual Google Assistant allows for interacting with Actions on Google programmatically.
+
+The core Virtual Google Assistant API provides several routines - the three most essential ones:
+
+    * launch: Generates JSON for a launch request
+    * utter: Generates JSON as if the user said the given phrase
+    * intend: Generates JSON as if the given intent was uttered
+
+## Why Do I Need This?
+This library allows for easy testing of actions on google.
+
+You can use it for:
+
+1) Unit-testing - ensuring individual routines work correctly
+2) Regression testing - ensuring the code as a whole works properly
+
+## How Do I Get It?
+```
+npm install virtual-google-assistant --save-dev
+```
+
+## How Do I Use It?
+Easy! Just add a line of code like so:
+```
+const vga = require("virtual-google-assistant");
+const assistant = vga.VirtualGoogleAssistant.Builder()
+    .handler("index.handler") // Google Cloud Function file and name
+    .directory("./dialogFlowFolder") // Path to the Dialog Flow exported model
+    .create();
+
+assistant.utter("play").then((payload) => {
+    console.log("OutputSpeech: " + result.speech);
+    // Prints out returned SSML, e.g., "<speak> Welcome to my Action </speak>"
+});
+```
+
+## Initializing the Virtual Google Assistant
+
+Currently we have three ways of setting up the virtual google assistant in order to work with your code.
+
+# Against an started server
+
+Just set the complete url and we will send the Requests payloads to it.
+
+```
+const vga = require("virtual-google-assistant");
+const assistant = vga.VirtualGoogleAssistant.Builder()
+    .actionUrl("http://myServer:3000/") // Example of a running server
+    .directory("./dialogFlowFolder") // Path to the Dialog Flow exported model
+    .create();
+```
+
+# Against a Google Cloud Function file.
+
+Set the path to the Action on Google code.
+
+```
+const vga = require("virtual-google-assistant");
+const assistant = vga.VirtualGoogleAssistant.Builder()
+    .handler("index.handler") // Google Cloud Function file and name
+    .directory("./dialogFlowFolder") // Path to the Dialog Flow exported model
+    .create();
+ ```
+
+
+# Against an express server
+
+Set the path to the file were your server is, along the port that you use.
+
+```
+const vga = require("virtual-google-assistant");
+const assistant = vga.VirtualGoogleAssistant.Builder()
+    .expressHandler("index", 3000) // Express server file and port
```

Comment:
I would label this expressModule and explain here that it must do `module.exports = server` to work. That's the trickiest thing about it.